### ISSUE TITLE
Avoid interpreting kepress '`' + 'i" as 'ì' instead of '`i' and going to the respective mark on the English International Keyboard on Windows

### ISSUE THREAD

**Issue by GoogleCodeExporter** [vim/vim#255]
```
I run gVim7.4 on Windows 7 with the English International keyboard. 

It configured in a way so that pressing '`' + 'i' gives you 'ì'. In Vim insert 
mode that's the desired behavior. However it isn't in command mode where I 
effectively have to press '`' + ' ' + 'i' to go to the mark 'i'.

I observed similar behavior for the caret '^' key. Pressing it makes vim wait 
for a vowel to be able to render 'â', 'î' etc instead of directly going to be 
beginning of the line. It only goes there when I press '^' + ' '. In insert 
mode it's useful to be able to type 'â'. It isn't in command mode and there 
it's completely useless to wait for another character. I thereby propose that 
vim looks directly for whether the '`' key or the '^' gets pressed and doesn't 
wait for a character.
```

Original issue reported on code.google.com by `christia...@gmail.com` on 8 Sep 2014 at 12:01

**Comment by 2072** [comment]
Note that up until patch 7.4.413 (#250) dead key support was unreliable.

I can reproduce this behaviour on 7.4.826: to actually go to mark **i** I need to press **`** then **space bar** and finally **i** (I got used to it). 

Since I have some experience with [Windows' dead keys management](https://msdn.microsoft.com/en-us/library/windows/desktop/ms646267%28v=vs.85%29.aspx#_win32_Dead_Character_Messages) working on patch 413 I'll take a look at this and see if I can make a PR with a fix.

**Comment by deragon** [comment]
If possible, it would be nice to have a solution for Linux too.  However, it would be nice that an option exist to enable / disable the solution, if ever you manage to have one.  It could be 'on' by default, but if someone is experiencing a problem with it, at least it could be turned off.

**Comment by 2072** [comment]
This is fixed by [patch 7.4.1124](https://github.com/vim/vim/commit/25b2b94ea73eff2aeef624d2ba7f59a1a265a0c1)
